packages/contracts/test/contracts/OVM/accounts/OVM_ECDSAContractAccount.spec.ts

```diff
@@ -190,4 +190,23 @@ describe('OVM_ECDSAContractAccount', () => {
       )
     })
   })
+  describe('isValidSignature()', () => {
+    // NOTE: There is no good way to unit test verifying a valid signature
+    // An integration test exists testing this instead
+    it(`should revert for a malformed signature`, async () => {
+      const messageHash = utils.keccak256('0x42')
```

Comment:
Message hash in the context here actually means more than just the keccak hash. You can use ethers `utils.hashMessage` helper for that https://docs.ethers.io/v5/api/utils/hashing/#utils-hashMessage